### PR TITLE
Joel's latest

### PR DESCRIPTION
**Pull request by amardesi**

### CODE REVIEW

File: app/jsx/components/MediaFileGridComp.jsx [hunk 1]
```diff
@@ -14,20 +22,16 @@ class CellComp extends React.Component {
...
       <div className={"o-mediafile--" + mimeSimple}>
-        {/* Some items have no titles but use descriptions instead, and these get moved into title.
-            Let's make these look better (and load faster) by avoiding dotdotdot in this case only.
-        */}
-        <h2 className="o-mediafile__heading" ref={ el => { if ((p.useFilenameForTitle) || (p.description && p.description.length > 0)) $(el).dotdotdot({watch:"window"}) } }>
-          <Link to={url} target="_blank">{p.title}</Link>
-        </h2>
-        <Link className="o-mediafile__link" to={url} aria-label={fileLabel} target="_blank">
+        <h2 className="o-mediafile__heading" ref={ el => { if (p.title && p.title.length > 38) $(el).dotdotdot({watch:"window"}) } }>{p.title}</h2>
+        <div className="o-mediafile__preview" onClick={() => {this.view(url)}} aria-label={fileLabel} target="_blank">
```
Comment (amardesi) [inline app/jsx/components/MediaFileGridComp.jsx]: This will eventually pop up a modal but Joel isn't done building it.

Comment (martinhaye) [reply app/jsx/components/MediaFileGridComp.jsx]: Ok

File: app/jsx/components/TabAuthorComp.jsx [hunk 2]
```diff
@@ -44,7 +44,8 @@ class TabAuthorComp extends React.Component {
       {!p.attrs['orig_citation'] &&
         <div className="c-itemactions">
           <div className="o-download">
-            <NotYetLink element="button" className="o-download__button">Download Citation</NotYetLink>
+            {/* ToDo: Link up citation */}
+            <a href="" className="o-download__button" download="">Download Citation</a>
```
Comment (martinhaye) [inline app/jsx/components/TabAuthorComp.jsx]: This should probably be a <NotYetLink> right?

File: app/jsx/objects/DownloadObj.jsx [hunk 3]
```diff
@@ -9,7 +9,7 @@ class DownloadObj extends React.Component {
         <h2>Using Single Menu Items</h2>
 
         <div className="o-download">
-          <button className="o-download__button">Download All Files</button>
+          <a href="" className="o-download__button" download>Download All Files</a>
```
Comment (martinhaye) [inline app/jsx/objects/DownloadObj.jsx]: ...likewise here...

File: app/jsx/objects/DownloadObj.jsx [hunk 4]
```diff
@@ -25,7 +25,7 @@ class DownloadObj extends React.Component {
         <h2>Using Nested Menu Items</h2>
 
         <div className="o-download">
-          <button className="o-download__button">Download PDF</button>
+          <a href="" className="o-download__button" download>Download PDF</a>
```
Comment (martinhaye) [inline app/jsx/objects/DownloadObj.jsx]: ...and here

File: app/jsx/components/TabAuthorComp.jsx (hunk 2 above)
Comment (amardesi) [reply app/jsx/components/TabAuthorComp.jsx]: @martinhaye  Not clear on your question.

Comment (martinhaye) [reply app/jsx/components/TabAuthorComp.jsx]: Oops, github edited out my tag. I think it should be a NotYetLink, right?

**Pull request merged**